### PR TITLE
SLING-11380: Upgrade to sling 47

### PR DESCRIPTION
**Pull request by apelluru**

### CODE REVIEW

File: src/test/java/org/apache/sling/jmx/provider/impl/JMXResourceProviderTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Dictionary;
+import java.util.Hashtable;
+
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(MockitoJUnitRunner.class)
```
Comment (rombert) [inline src/test/java/org/apache/sling/jmx/provider/impl/JMXResourceProviderTest.java]: Why do you need the Mockito runner and the mockito dependency?

Comment (apelluru) [reply src/test/java/org/apache/sling/jmx/provider/impl/JMXResourceProviderTest.java]: Its not required, I have removed mockito dependency

File: pom.xml [hunk 2]
```diff
@@ -22,44 +22,28 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.apache.sling</groupId>
-        <artifactId>sling</artifactId>
-        <version>26</version>
+        <artifactId>sling-bundle-parent</artifactId>
+        <version>47</version>
```
Comment (kwin) [inline pom.xml]: The most recent version is 48 meanwhile

File: pom.xml [hunk 3]
```diff
@@ -22,44 +22,28 @@
...
     <version>1.0.3-SNAPSHOT</version>
-    <packaging>bundle</packaging>
 
     <name>Apache Sling JMX Resource Provider</name>
 
     <scm>
         <connection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-jmx-provider.git</connection>
         <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-jmx-provider.git</developerConnection>
         <url>https://gitbox.apache.org/repos/asf?p=sling-org-apache-sling-jmx-provider.git</url>
+        <tag>HEAD</tag>
```
Comment (kwin) [inline pom.xml]: This is the default value which doesn’t need to be set explicitly

File: pom.xml [hunk 4]
```diff
@@ -68,27 +52,30 @@
...
-            <scope>provided</scope>
-        </dependency>
         <dependency>
             <groupId>org.osgi</groupId>
-            <artifactId>org.osgi.compendium</artifactId>
-        </dependency>
-        <dependency>
-        	<groupId>org.apache.felix</groupId>
-        	<artifactId>org.apache.felix.scr.annotations</artifactId>
+            <artifactId>osgi.cmpn</artifactId>
```
Comment (kwin) [inline pom.xml]: Please switch to individual OSGi dependencies (https://docs.osgi.org/artifacts/)

File: pom.xml (hunk 2 above)
Comment (apelluru) [reply pom.xml]: updated to 48

File: pom.xml (hunk 3 above)
Comment (apelluru) [reply pom.xml]: removed 👍

File: pom.xml (hunk 4 above)
Comment (apelluru) [reply pom.xml]: Fixed 👍

**Review by rombert — APPROVED** [review]
LGTM, thanks

**Review by kwin — APPROVED** [review]
Thanks for incorporating my feedback

**Pull request merged**